frontend/src/utils/validations.ts

```diff
@@ -26,21 +26,80 @@ export function validateEmail(value: string): ValidationResult {
   };
 }
 
+//isEightCharsLong checks if password is at least eight characters
+function isEightCharsLong(value: string): boolean {
+  if (value.length >= 8) return true;
+  return false;
+}
+
+//containsOneLetter checks if password contains at least one letter
+function containsOneLetter(value: string): boolean {
+  const regexPattern = /[A-Za-z]/;
```

Comment:
Let's move this content to the top of he file and rename it to `R_CONTAIN_ONE_LETTER`